freie-universitat-berlin-geographische-wissenschaften.csl

```diff
@@ -180,10 +188,8 @@
       <key macro="author"/>
     </sort>
     <layout prefix="(" suffix=")" delimiter="; ">
-      <group delimiter=" ">
-        <text macro="author-short"/>
-        <text macro="year-date"/>
-      </group>
+      <text macro="author-short" suffix=" "/>
+      <text macro="year-date"/>
```

Comment:
Assume "year-date" is missing, then we will still have the space from the suffix in your current code.